_posts/2024-04-25-guardado-automatico-propuesta-solicitud-linea.md

```
@@ -0,0 +1,23 @@
+author: Catalina Pinilla
+---
+
+## Propuesta de negocio - Solicitud de línea
+
+### Auto guardado
+
+Ya no debes preocuparte por olvidar un paso, ahora la `Presentación de cliente ` en la `Propuesta` de `Solicitud de línea` se guarda de manera automática para no sufrir accidentes cuando haya intermitencias de red, recargues la página o vuelvas atrás accidentalmente ya no será un problema, porque implementamos el guardado automático para que lo que redactes en la `Presentación de cliente` siempre quede respaldado.
+Pero, ¿Cómo te podemos asegurar que tu trabajo esta guardado?
+Fácil, cada vez que escribes verás un ticket que significa que tu proceso fue guardado con éxito.
+
+- <video width="100%" controls>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: video

(MD033, no-inline-html)